scheduler/views.py

```diff
@@ -14,12 +13,22 @@
 from django.views.decorators.cache import never_cache
 from redis.exceptions import ResponseError
 
-from .queues import get_all_workers, get_connection, QueueNotFoundError
+
+from .queues import get_all_workers, get_connection, QueueNotFoundError, ConnectionErrors
 from .queues import get_queue as get_queue_base
 from .rq_classes import JobExecution, DjangoWorker, DjangoQueue, InvalidJobOperation
 from .settings import SCHEDULER_CONFIG, logger
 
 
+try:
+    from valkey import exceptions
+except ImportError:
+    exceptions = ""
```

Comment:
Perhaps move `ResponseErrors` and `ConnectionErrors` to `connection_types.py`? (and maybe rename the file to `broker_typings.py`)